tests/functional/utils/casper-setup.js

```diff
@@ -0,0 +1,26 @@
+/*globals casper */
+
+'use strict';
+
+casper.userAgent('Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:35.0) Gecko/20100101 Firefox/35.0');
+
+casper.host = 'http://' + casper.cli.options.host;
+
+casper.on('page.initialized', function () {
+    casper.page.injectJs('tests/functional/includes/fn.bind.polyfill.js');
+    casper.page.injectJs('tests/functional/includes/html.event.polyfill.js');
```

Comment:
In theory, we could. It just means that we'd have to request the fn.bind polyfill, which is needed by the site anyway. Not sure if there is a polyfill for html events though...